### DISCUSSION TITLE
Random requests can ruin users authentication

### DISCUSSION THREAD

**Discussion by palaxius** [nuxt/nuxt#23419] [Questions]
Hello,

Nuxt 3 Project. I have several requests in the nuxt 3 plugin that are processed until the application is fully initialized, including the refresh method on the backend, which updates the JWT tokens.

The problem is if some request occurs inside the application for a broken url (due to the fault of the backend, frontend or someone else), for example site.com/bluh-bluh-bluh -- all plugins will fully work out and will update my tokens on the backend and my current session will not be valid. What can be done about it?

Thanks in advance.

**Comment by cwe-dev** [comment]
@pi0 hi. this question is very important to me, as it strongly blocks the logic of the site. turning off ssr is not a solution.
